### PR TITLE
Added Kernel version

### PR DESCRIPTION
**Pull request by theSuess**
Added the kernel version, currently only on linux

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -52,6 +52,14 @@ global.weather = function(){
     });
 };
 
+global.uname = function(){
+    exec('uname -a',function(err,stdout,stderr) {
+        if(err) throw err;
+        if(stdout) {
+            io.emit('uname',stdout.toString('utf8').split('#')[0].split(' ')[2]);
```
Comment (wbrbr) [inline server.js]: The`uname -r` command does what you want, and is simpler

**Comment by wbrbr** [comment]
Thanks for your contribution :smiley:

You should add an entry in the `modules` array in `config.json`

``` json
{
    name:"uname",
    interval:100000
}
```

**Comment by theSuess** [comment]
Yeah sorry it was late and I wanted to code something :D anyways I fixed it :)

**Comment by wbrbr** [comment]
Haha no problem :P

**Pull request merged**